### ISSUE TITLE
EPEL Wishlist 

### ISSUE THREAD

**Issue by hanthor** [castrojo/achillobator#59]
@carlwgeorge has already been working away at some requests here:
https://copr.fedorainfracloud.org/coprs/carlwgeorge/achillobator-incubator/

- [x] just
- [x] fastfetch
- [x] gnome-teaks
- [x] blur-my-shell
- [x] fzf
- [ ] restic
- [ ] rclone
- [x] wl-clipboard
- [x] jetbrains-mono-font

**Comment by hanthor** [comment]
We'd love fzf it's just not in EPEL 10 yet

**Comment by hanthor** [comment]
We probably don't need or want most of these but here's all the packages we isntall in bluefin that aren't avaialbe in dnf with EPEL and CBR on

 ## Packages we install in Bluefin(-dx) that aren't currently avaiable in EPEL 10
 - adw-gtk3-theme 
 - bcache-tools 
 - borgbackup
 - evtest 
 - epson-inkjet-printer-escpr 
 - epson-inkjet-printer-escpr2 
 - foo2zjs
 - fuse-encfs 
 - gum 
 - glow
 - input-leap 
 - input-remapper 
 - jetbrains-mono-fonts-all 
 - opendyslexic-fonts 
 - printer-driver-brlaser 
 - rclone 
 - restic 
 - solaar 
 - wl-clipboard

**Comment by tulilirockz** [comment]
I believe we can get away with not adding most of these for now tho

**Comment by hanthor** [comment]
we are already adding glow from charm. we can get it added to EPEL instead?

**Comment by tulilirockz** [comment]
i think we should be just a bit more conservative with this wishlist for now LOL. a good target for now would be to have parity with bluefin base if possible. there might also be some things we dont need here

**Comment by carlwgeorge** [comment]
Assuming all of these are open source, they should all be possible to add to EPEL.  Things that are already in Fedora are easier.  What would help me is a ranked order of importance so I know where to focus my time first.

**Comment by castrojo** [comment]
I don't think we want to blanket ask for these as the criteria for support would be longer. Printers and fonts will require some thought. 

Restic and rclone are nobrainers, let's go with those for an easy win?

**Comment by tulilirockz** [comment]
I personally would love mkosi on epel so that we can develop sysexts and initramfs for this with the systemd centric tooling

**Comment by tulilirockz** [comment]
This is our current default font on Bluefin, this might be super cool to have on EPEL too

https://packages.fedoraproject.org/pkgs/jetbrains-mono-fonts/jetbrains-mono-fonts/

**Comment by hanthor** [comment]
https://packages.fedoraproject.org/pkgs/wl-clipboard/

wl-clipboard is real nice thing to have. copying out of tmux is a pain

**Comment by carlwgeorge** [comment]
Completed requests (so far):

- [fastfetch](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2024-de05cb8bd2)
- [just](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2024-f017ddda07)
- [blur-my-shell](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2024-d60842a3f0)

**Comment by tulilirockz** [comment]
@carlwgeorge Theres a HackMD that we have that has some of the packages we don't have here - Might be worth checking it out? https://hackmd.io/T2OAjyXIS12kZ2Ljz2q_0g?edit=

**Comment by tulilirockz** [comment]
Conda and Ollama would be amazing too!

**Comment by bsherman** [comment]
Here's some packages which ARE in EPEL **9** but NOT in EPEL **10**, so I'd expect this to be obvious for inclusion in EPEL 10... 

- p7zip
- p7zip-plugins
- picocom
- unrar

Do we have a process for that or is it a generic request upstream?

**Comment by bsherman** [comment]
Process for packages we want added... 

https://docs.fedoraproject.org/en-US/epel/epel-package-request/

**Comment by carlwgeorge** [comment]
[p7zip (and p7zip-plugins) has been requested](https://bugzilla.redhat.com/show_bug.cgi?id=2315312), but it sounds like the maintainer would rather add 7zip instead, which it seems is the currently maintained fork (or something like that).  I don't see any open requests for the others.

**Comment by bsherman** [comment]
[picocom EPEL 10 request](https://bugzilla.redhat.com/show_bug.cgi?id=2348497) has been filed.

**Comment by bsherman** [comment]
[unrar-free EPEL 10 request](https://bugzilla.redhat.com/show_bug.cgi?id=2348499) has been filed.

edit: this has been completed.

**Comment by bsherman** [comment]
[required for akmods / kernel-cache builds](https://github.com/ublue-os/akmods/issues/305)

[rpmrebuild EPEL 10 request](https://bugzilla.redhat.com/show_bug.cgi?id=2358309) has been filed

**Comment by bsherman** [comment]
[required for akmods / kernel-cache builds](https://github.com/ublue-os/akmods/issues/305)

[sbsigntools EPEL 10 request](https://bugzilla.redhat.com/show_bug.cgi?id=2358317) has been filed

**Comment by carlwgeorge** [comment]
[rpmrebuild-2.20-3.el10_1](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2025-afe1aa9eb2) and [sbsigntools-0.9.5-10.el10_1](https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2025-768b1bfee3) are now in epel-testing.

**Comment by hanthor** [comment]
restic/rclone may be blocked on https://discussion.fedoraproject.org/t/f43-change-proposal-golang-packages-vendored-by-default-system-wide/154986

**Comment by jfmongrain** [comment]
evince-thumbnailer would be nice to have feature parity with Bluefin
For now, pdf files don't have thumbnails in Nautilus.

**Comment by gbrsni** [comment]
[epson-inkjet-printer-escpr EPEL 10 request](https://bugzilla.redhat.com/show_bug.cgi?id=2392988) has been filed
